pyccl/tk3d.py

```diff
@@ -45,52 +45,55 @@ class Tk3D(CCLObject):
+            and ``nk`` are the sizes of ``a_arr`` and ``lk_arr`` respectively.
             This array should contain the values of the trispectrum
-            at the values of scale factor and wavenumber held by `a_arr`
-            and `lk_arr`. The array can hold the values of the natural
+            at the values of scale factor and wavenumber held by ``a_arr``
+            and ``lk_arr``. The array can hold the values of the natural
             logarithm of the trispectrum, depending on the value of
-            `is_logt`. If `tkk_arr` is `None`, then it is assumed that
+            ``is_logt``. If ``tkk_arr`` is ``None``, then it is assumed that
             the trispectrum can be factorized as described above, and
             the two functions :math:`f_i(k_i,a)` are described by
-            `pk1_arr` and `pk2_arr`. You are responsible for making sure
+            ``pk1_arr`` and ``pk2_arr``. You are responsible for making sure
             all these arrays are sufficiently well sampled (i.e. the
-            resolution of `a_arr` and `lk_arr` is high enough to sample
+            resolution of ``a_arr`` and ``lk_arr`` is high enough to sample
             the main features in the trispectrum). For reference, CCL
             will use bicubic interpolation to evaluate the trispectrum
             in the 2D space of wavenumbers :math:`(k_1,k_2)` at a fixed
             scale factor, and will use linear interpolation in the
             scale factor dimension.
-        pk1_arr (array): a 2D array with shape `[na,nk]` describing the
+        pk1_arr (array): a 2D array with shape ``[na,nk]`` describing the
             first function :math:`f_1(k,a)` that makes up a factorizable
             trispectrum :math:`T(k_1,k_2,a)=f_1(k_1,a)f_2(k_2,a)`.
-            `pk1_arr` and `pk2_arr` are ignored if `tkk_arr` is not
-            `None`.
-        pk2_arr (array): a 2D array with shape `[na,nk]` describing the
+            ``pk1_arr`` and ``pk2_arr`` are ignored if ``tkk_arr`` is not
+            ``None``.
+        pk2_arr (array): a 2D array with shape ``[na,nk]`` describing the
             second factor :math:`f_2(k,a)` for a factorizable trispectrum.
-        is_logt (boolean): if True, `tkk_arr`/`pk1_arr`/`pk2_arr` hold the
-            natural logarithm of the trispectrum (or its factors).
+        is_logt (boolean): if True, ``tkk_arr``/``pk1_arr``/``pk2_arr``
+            hold the natural logarithm of the trispectrum (or its factors).
             Otherwise, the true values of the corresponding quantities are
             expected. Note that arrays will be interpolated in log space
-            if `is_logt` is set to `True`.
+            if ``is_logt`` is set to ``True``.
         extrap_order_lok (int): extrapolation order to be used on k-values
             below the minimum of the splines (use 0 or 1). Note that
             the extrapolation will be done in either
             :math:`\\log(T(k_1,k_2,a)` or :math:`T(k_1,k_2,a)`,
```

Comment:
Missing closing parenthesis. Also, below `ln` is used instead of `log`.

Reply:
Done. Used `log` throughout.